shared/more/dumb-sheet.render.native.js

```diff
@@ -4,7 +4,6 @@ import {ScrollView} from 'react-native'
 import {Box, Text, Input, Button} from '../common-adapters'
 import {globalStyles} from '../styles/style-guide'
 import dumbComponentMap from './dumb-component-map.native'
-
```

Comment:
Was this newline removal intended? The desktop variant of this file has a newline here as well -- if this was a cleanup, perhaps remove it too?